theme/tuxlite_zf/templates/index.html

```diff
@@ -22,6 +22,9 @@ <h2>Upcoming</h2>
 <h3><a href="https://scipy2024.scipy.org/">SciPy 2024</a></h3>
 <p>Tacoma, WA, July 8-14, 2024</p>
 
+<h3><a href="https://euroscipy.org/2025/">EuroSciPy 2025</a></h3>
+<p>Krakow, Poland, August 18-22, 2025</p>
+
```

Comment:
I'd just opened #65, didn't notice this PR was already open. And this one is better, as you caught some things (like the proceedings link) that I'd missed.

Could you include the one change from there that I don't see here? SciPy 2024 is no longer "Upcoming", so it should be replaced on this page with

```text
<h3><a href="https://scipy2025.scipy.org/">SciPy 2025</a></h3>
<p>Tacoma, WA, July 7-13, 2025</p>
```

Reply:
ha there is yet ANOTHER PR already doing that 🤦🏻 : #63

I really should have read the open list of PRs.